GVRf/Framework/jni/shaders/posteffect/custom_post_effect_shader.cpp

```diff
@@ -137,6 +139,16 @@ void CustomPostEffectShader::render(
         glUniform1i(u_texture_, texture_index++);
     }
 
+    if (u_projection_matrix_ != -1) {
+        glm::mat4 view = camera->getViewMatrix();
+        glm::mat4 proj = glm::mat4(
```

Comment:
Hi,

On Wed, May 13, 2015 at 10:45:52AM -0700, thomasflynn wrote:
> > @@ -137,6 +139,16 @@ void CustomPostEffectShader::render(
> >          glUniform1i(u_texture_, texture_index++);
> >      }
> >  
> > +    if (u_projection_matrix_ != -1) {
> > +        glm::mat4 view = camera->getViewMatrix();
> > +        glm::mat4 proj = glm::mat4(
> 
> that's what i suspected, but couldn't you just pass GL_TRUE to glUniformMatrix?
> 

Indeed, I'll update this and post the updated patch.
Thanks